dts/common/nordic/nrf54h20.dtsi

```diff
@@ -8,13 +8,19 @@
 #include <nordic/nrf_common.dtsi>
 
 #include <zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20.h>
+#include <zephyr/dt-bindings/reserved-memory/nordic-owned-memory.h>
 
 /* Domain IDs. Can be used to specify channel links in IPCT nodes. */
 #define NRF_DOMAIN_ID_APPLICATION 2
 #define NRF_DOMAIN_ID_RADIOCORE   3
 #define NRF_DOMAIN_ID_GLOBALFAST  12
 #define NRF_DOMAIN_ID_GLOBALSLOW  13
 
+/* Owner IDs. Can be used to specify memory access. */
+#define NRF_OWNER_ID_NONE        0
+#define NRF_OWNER_ID_APPLICATION 2
+#define NRF_OWNER_ID_RADIOCORE   3
```

Comment:
Those magic numbers should either be validated against values in MDK or generated from SVD.

Reply:
@57300 I think @gmarull has a script to generate from SVD that you could use here.

Reply:
@carlescufi That's true, but these values aren't found in SVD. I will add MDK validation in another PR.

Reply:
@57300 would you mind adding the MDK validation directly here?

Reply:
> @57300 would you mind adding the MDK validation directly here?

I'm still waiting for other dependency PRs to get merged before I rebase.
In the meantime, I opened a new PR for this (you'll see why): https://github.com/zephyrproject-rtos/zephyr/pull/72507